omnibus/config/software/oc_bifrost.rb

```diff
@@ -25,7 +25,7 @@
 
 build do
   env = with_standard_compiler_flags(with_embedded_path)
-  profile_name = fips_mode? ? "fips" : "default"
+  profile_name = "default"
```

Comment:
Not important right now, but we can probably remove the profile_name stuff completely and just assume "default" which is what rebar3 will build by...default.